lightningd/payalgo.c

```diff
@@ -612,6 +618,7 @@ static void json_pay(struct command *cmd,
 		   p_opt_def("retry_for", json_tok_number, &retryfor, 60),
 		   p_opt_def("maxdelay", json_tok_number, &maxdelay,
 			     cmd->ld->config.locktime_max),
+		   p_opt_def("exemptfee", json_tok_number, &exemptfee, 5000),
```

Comment:
does that also make the argument available as an option in the config file?

Reply:
No. This only deals with json parameters. `config` and command line parameters are handled in `lightningd/options.c`.